frontend/packages/ceph-storage-plugin/src/utils/odf-provisioners.ts

```diff
@@ -0,0 +1,9 @@
+import { ProvisionerDetails } from 'packages/console-dynamic-plugin-sdk';
+
+type Parameters = ProvisionerDetails['parameters'];
+
+export const getTrue = () => true;
+export const getFalse = () => false;
```

Comment:
The rationale behind doing this is to not have to perform type checking at runtime. Although it should be possible to have a union of function and a boolean value.

Reply:
> The rationale behind doing this is to not have to perform type checking at runtime.

This doesn't seem too difficult? It would make the API much easier to use for plugins.